about.html

```diff
@@ -0,0 +1,146 @@
+          <li class="nav-item"><a href="#" class="nav-link">My Page</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Logout</a></li>
+        </ul>
+      </div>
+      <div class="header-bottom">
+        <a href="./index.html" class="logo"><img src="./images/elearning-logo.png" alt="logo" />
+        </a>
+        <div class="nav-right">
+          <ul class="main-nav">
+            <li class="nav-item">
+              <a href="./index.html" class="nav-link">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="./index.html#program" class="nav-link">Program</a>
+            </li>
+            <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+            <li class="nav-item">
+              <a href="./index.html#sponsor" class="nav-link">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-link campain">CG Campain</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <main class="about-wrapper">
+    <section class="about">
+      <div class="about-us">
+        <span class="about-quote">"Learning experiences are like journeys.
+          The journey starts where the learning is now, and ends when the learner is successful.
+          The end of the journey isn't knowing more, it's doing more."</span>
+        <h1 class="about-goal">
+          “Meet the learning style or needs of students.
+          Improve the efficiency and effectiveness.
+          Improve user-accessibility and time”
+        </h1>
+      </div>
+      <div class="about-ideal">
+        E-Learning is the learning facilitated and supported by Information Communication technologies
+        (ICT) to enable people to learn anytime and anywhere.
+        E-learning is nothing but the use of technology to connect.
+        teachers and students who are physically miles apart.
+        Learning objectives are the cornerstone of every e-learning course. They’re the reason you’re
+        creating the course. They guide you as you select the content and activities to include.
+        And they help you determine whether your course has been effective.
+        So basically, without learning objectives you won’t know why you’re creating the
+        course, what content to include, what activities to choose, or how to gauge the success of your course.
+        Seems pretty important when you put it that way, doesn’t it?
+      </div>
+      <div class="contact">
+        <p>
+          Please contact us per Email for any further questions about E-learning .
+        </p>
+        <p>
+          <strong><a href="ibouaik@gmail.com">ibouaik@gmail.com</a></strong>
```

Comment:
- [ ] Kindly try to match the Behance design in the intro section on the home page mobile version by increasing the `font-size` of the heading and making the text in the center

| Your design | Behance design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/164566511-422eaa75-3573-446b-a053-2f1d541f990b.png) | ![image](https://user-images.githubusercontent.com/24703171/164566532-46303f9c-41fc-4d73-86d7-7c918a2437a4.png) |

- [ ] and also the same in the desktop version